README.md

```diff
@@ -384,6 +385,7 @@ Engines, libraries and other helpful things specifically for making games.
   all open source.
 * [Spearmint][371] - Engine designed for FPS games. [``GPL-3.0-or-later``][GPL-3.0-or-later]
 * [uastar][535] - Minimal A\* implementation. [``ZLib``][Zlib]
+* [cglm](https://github.com/recp/cglm) - 📽 Optimized OpenGL/Graphics Math (glm) for C. [MIT](https://github.com/recp/cglm/blob/master/LICENSE)
```

Comment:
This looks good, aside from inline link use and non-alphabetical ordering. Please fix these.